resources/migrations/001_update_migrations.yaml

```diff
@@ -8901,8 +8901,14 @@ databaseChangeLog:
 
   - changeSet:
       id: v51.2024-07-25T12:02:21
+      validCheckSum: ANY
       author: crisptrutski
       comment: Make `field_id` nullable on query fields
+      rollback:
+        - delete:
+            tableName: query_field
+            where: query_field.field_id is NULL
+            remarks: remove stale analysis, so the fields can be non-nullable
```

Comment:
Hmm this overrides bringing back the constraint.

Don't see the hard in that though. Maybe we don't even need to delete these rows, but I don't see the harm there either.